utils/filter.c

```diff
@@ -113,8 +113,7 @@ struct uftrace_filter *uftrace_match_filter(uint64_t ip, struct rb_root *root,
 		iter = rb_entry(parent, struct uftrace_filter, node);
 
 		if (match_ip(iter, ip)) {
-			memcpy(tr, &iter->trigger, sizeof(*tr));
-
+			*tr = iter->trigger;
```

Comment:
Please add a commit message to describe **WHY** this is needed.  Maybe you can quote (a part of) compiler error message and explanation.

Reply:
ok